### ISSUE TITLE
CKEditor - unable to add custom styles

### ISSUE THREAD

**Issue by Graham-72** [backdrop/backdrop-issues#2705]
I have added custom styles to ckeditor-iframe.css and added this css file to my theme but it is not being applied by the preview iframe.
I have also tried editing ckeditor-iframe.css in the core ckeditor module but this has no effect either.

Has anybody got this feature working? If so, how?

I am using 1.7.0

**Comment by Al-Rozhkov** [comment]
Yes, it is works for me. I added `ckeditor_stylesheets[] = css/ckeditor-iframe.css` to my theme info file.

**Comment by Graham-72** [comment]
@Al-Rozhkov thank you. That solves my problem!

It was the `ckeditor_stylesheets[] =` I needed. I had `stylesheets[all][] =`

Is this documented in Backdrop somewhere?

**Comment by Al-Rozhkov** [comment]
I'm not sure. I remember it was not easy to find :smile:

**Comment by quicksketch** [comment]
Documentation is currently only mentioned in these functions:

https://api.backdropcms.org/api/backdrop/core%21modules%21ckeditor%21ckeditor.module/function/_ckeditor_theme_css/1
https://api.backdropcms.org/api/backdrop/core%21modules%21ckeditor%21ckeditor.api.php/function/hook_ckeditor_css_alter/1

And it's in the UI (as updated by Graham's patch). I'm not sure about putting theme-development documentation directly in the UI. Perhaps we update and link to the development guide for themes? https://api.backdropcms.org/themes

**Comment by quicksketch** [comment]
I added a section for `ckeditor_styles[]` to https://api.backdropcms.org/themes.

We do need to update the documentation in the CKEditor module (similar to the current PR). The current docs themselves aren't accurate because the name of the file isn't necessarily `ckeditor-iframe.css` in the first place.

**Comment by klonos** [comment]
There was only a short while ago another question about how one can switch CKEditor theme per Backdrop theme. Is that currently possible? If so, perhaps it is also worth mentioning how in the dev/theming docs.

**Comment by quicksketch** [comment]
Whatever the front-end theme is, the matching CSS (`ckeditor_styles[]`) from that theme's .info file will be loaded. There isn't any ability to have multiple different stylesheets loaded per theme. A module could be used that leverages `hook_ckeditor_css_alter()` to accomplish this, but a theme alone could not do this because the front-end theme isn't loaded when on administrative pages.
